openml/flows/flow.py

```diff
@@ -105,7 +377,7 @@ def _ensure_flow_exists(self):
         """
         import sklearn
         flow_version = 'sklearn_' + sklearn.__version__
-        _, _, flow_id = _check_flow_exists(self._get_name(), flow_version)
+        _, _, flow_id = _check_flow_exists(self.name, flow_version)
```

Comment:
I get an error in the tests in line 384. publish returns self which is not iterable....